### PR TITLE
Update of the bids-starter-kit documentation for PET

### PR DESCRIPTION
**Pull request by melanieganz**
- this update only changes the bids-starter-kit documentation for PET
- it showcases the usage of PET2BIDS tools for Python and MatLab for conversion as well as includes links to phantom data that has source and converted data

### CODE REVIEW

File: src/tutorials/pet.md [hunk 1]
```diff
@@ -2,43 +2,180 @@
...
+The PET modality is a recent addition to BIDS with its introduction via BEP 009. If you're interested in seeing exactly what and how something gets added to BIDS see the pull request for BEP009 [here](https://github.com/bids-standard/bids-specification/pull/633). The results of that extension proposal can be read [here](https://bids-specification.readthedocs.io/en/stable/04-modality-specific-files/09-positron-emission-tomography.html#positron-emission-tomography) in the bids standard.
+
+## PET image data file formats
+
+
+Before we start to convert data we need to quickly mention that PET image data files come of the scanner in various different formats, some scanners provide DICOM files (.dcm) and others use proprietary formats e.g. ECAT format (.v) . In order to facilitate easy testing of data conversion across different PET file formats  the [OpenNeuroPET project](https://openneuropet.github.io/) has compiled a bunch of phantom data from different scanner types and is distributing two examples [here](https://drive.google.com/file/d/10S0H7HAnMmxHNpZLlifR14ykIuiXcBAD/view?usp=sharing) . You can download them for testing purposes either manually or in the terminal by typing
```
Comment (Remi-Gau) [inline src/tutorials/pet.md:10]:
```suggestion
Before we start to convert data we need to quickly mention that PET image data files come of the scanner in various different formats, some scanners provide DICOM files (.dcm) and others use proprietary formats for example ECAT format (.v) . In order to facilitate easy testing of data conversion across different PET file formats  the [OpenNeuroPET project](https://openneuropet.github.io/) has compiled a bunch of phantom data from different scanner types and is distributing two examples [here](https://drive.google.com/file/d/10S0H7HAnMmxHNpZLlifR14ykIuiXcBAD/view?usp=sharing) . You can download them for testing purposes either manually or in the terminal by typing 
```

Comment (Remi-Gau) [reply src/tutorials/pet.md:10]: just to pacify the "bad latin" github action

**Comment by melanieganz** [comment]
@Remi-Gau, are you happy with this?

**Comment by Remi-Gau** [comment]
Sorry. Started to review but I have not finished.
Was planning to go through the whole thing playing the "naive user" making sure it runs smoothly.

**Comment by melanieganz** [comment]
Ah, that is very much appreciated. 👏  No worries, I know how busy you are. I was wonderign if you wanted me to do it differently. I will use part of this in a tutorial at Neurohackademy this week where we might also get some feedback. 👍

File: src/tutorials/pet.md [hunk 2]
```diff
@@ -198,16 +322,11 @@ NewBidsDataSet/
...
 
-**Note:** if you have multiple PET image files you can distinguish between them
-by creating a session folder with a unique name and then applying the
-`ses-<label>` label to each file therein. If there's a single pet scan you may
-omit the additional folder and corresponding label(s) from the filename. For
-more information on labeling see
-[this link](https://bids-specification.readthedocs.io/en/stable/04-modality-specific-files/09-positron-emission-tomography.html#pet-recording-data).
+**Note:** if you have multiple PET image files you can distinguish between them by creating a session folder with a unique name and then applying the
+`ses-<label>` label to each file therein. If there's a single pet scan you may omit the additional folder and corresponding label(s) from the filename. For
+more information on labeling see [this link](https://bids-specification.readthedocs.io/en/stable/04-modality-specific-files/09-positron-emission-tomography.html#pet-recording-data).
```
Comment (Remi-Gau) [inline src/tutorials/pet.md:329]:
```suggestion
```{Note}
If you have multiple PET image files you can distinguish between them by creating a session folder with a unique name
and then applying the `ses-<label>` label to each file therein. 
If there's a single pet scan you may omit the additional folder and corresponding label(s) from the filename.
For more information on labeling see [this link](https://bids-specification.readthedocs.io/en/stable/04-modality-specific-files/09-positron-emission-tomography.html#pet-recording-data).
```
```

**Review by Remi-Gau — APPROVED** [review]
Looks good to me.

Will push some additional commits after I have checked how it builds locally.

Most likely just some reformating

**Pull request merged**